backend/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
-
```

Comment:
Please add this line
"spring.h2.console.enabled=true"
so that we can access a UI for the db at "http://localhost:8080/h2-console";

Maybe add some comments as well?

Reply:
Ok! Will do